Userland/Services/ImageDecoder/ConnectionFromClient.cpp

```diff
@@ -54,6 +54,8 @@ static void decode_image_to_details(Core::AnonymousBuffer const& encoded_buffer,
         dbgln_if(IMAGE_DECODER_DEBUG, "Could not decode image from encoded data");
         return;
     }
+    is_animated = decoder->is_animated();
+    loop_count = decoder->loop_count();
```

Comment:
We would have caught this issue sooner if we didn't have these two lines above:
```
    VERIFY(!is_animated);
    VERIFY(loop_count == 0);
```

I see no reason to `VERIFY` these conditions since the defaults are not governed by this function, but by its invoker. Maybe just remove these two statements?

Reply:
I've updated the commit to also remove these lines. is that okay?